lighthouse/classes/event_properties/definitions.py

```diff
@@ -0,0 +1,213 @@
+from functools import cached_property
+from lighthouse.classes.messages.warehouse_messages import (  # type: ignore
+    ROLE_TYPE_CP_SOURCE_LABWARE,
+    SUBJECT_TYPE_PLATE,
+    ROLE_TYPE_ROBOT,
+    SUBJECT_TYPE_ROBOT,
+)
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract, RetrievalError  # type: ignore
+from lighthouse.classes.event_properties.validations import SimpleEventPropertyMixin  # type: ignore
+from typing import Any, List, Dict
+from lighthouse.classes.services.cherrytrack import ServiceCherrytrackMixin  # type: ignore
+from lighthouse.classes.services.mongo import ServiceMongoMixin  # type: ignore
+from lighthouse.constants.fields import (
+    FIELD_CHERRYTRACK_LH_SAMPLE_UUID,
+    FIELD_EVENT_RUN_ID,
+    FIELD_EVENT_ROBOT,
+    FIELD_EVENT_USER_ID,
+    FIELD_EVENT_BARCODE,
+)
+
+from flask import current_app as app
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RunID(EventPropertyAbstract, SimpleEventPropertyMixin):
+    def validate(self):
```

Comment:
If this is returning a bool, implying whether the parameter is valid or not, I think it is clearer to call it `is_valid` or something like that?

Reply:
Agree, I'll change it

Reply:
Done